docs/api.md

```diff
@@ -988,6 +1042,42 @@ The size of the attached terminal.
 
 ---
 
+## `ExportOptions`
+
+Options that control exporting data.
+
+```ts
+export interface ExportOptions {
+  format?: 'json' | 'binary' | 'zip',
+  includes?: string[];
+  excludes?: string[];
+}
+```
+
+### `ExportOptions` Properties
+
+<br />
+
+#### ▸ `format?: 'json' | 'binary' | 'zip'`
+
+The format of the exported data. The `json` and `binary` format can be used as `tree` when calling [`mount`](#▸-mount).
+
+The default value is `json`.
+
+<br />
+
+#### ▸ `includes?: string[]`
+
+Globbing patterns to include files from within excluded folders.
```

Comment:
Oh gosh I had to read this twice to understand it :face_with_spiral_eyes: 

I don't think we can make it simpler though :smile:

Reply:
Yeah I know 😂 .